docs/semgrep-ci/configuration-reference.md

```diff
@@ -19,6 +19,54 @@ While environment variables are the preferred way to configure Semgrep CI, pass
 SEMGREP_RULES="p/security-audit p/secrets"
 ```
 
+## Suppressing blocking comments or errors
+
+Semgrep in CI can block a pull request (PR) or a merge request (MR) from being merged. This block can be caused by Semgrep's blocking comment or Semgrep's in CI internal error. One option to avoid blocked PRs and MRs is to change configuration of blocking comments in the rule board of Semgrep App. Another option that Semgrep in CI allows you is to disable blocking comments without changing the setup of rules in the rule board.
```

Comment:
Can you review this "Semgrep's in CI internal error" phrase? I assume you meant to write just "Semgrep's internal error"?